spice_agent/inference/actions.py

```diff
@@ -185,7 +331,16 @@ def run_pipeline(
             elif is_text_input and is_file_output:
                 SPICE_INFERENCE_DIRECTORY.mkdir(parents=True, exist_ok=True)
                 save_at = Path(SPICE_INFERENCE_DIRECTORY / f"{inference_job_id}.png")
+
                 stable_diffusion_options = self._get_stable_diffusion_options(options)
+                prompt = text_input
+                negative_prompt = ""
+                if "negative_prompt" in stable_diffusion_options:
+                    negative_prompt = stable_diffusion_options["negative_prompt"]
```

Comment:
may be useful to use walrus
```suggestion
                if negative_prompt := stable_diffusion_options['negative_prompt"]:
```